### PR TITLE
OpenTelemetry operator docs fixes and improvements

### PR DESCRIPTION
**Pull request by prashant-shahi**
Signed-off-by: Prashant Shahi <prashant@signoz.io>

### CODE REVIEW

File: docs/tutorial/instrumentation-using-opentelemetry-operator.md [hunk 1]
```diff
@@ -14,22 +14,23 @@ import OtelOperatorCleanUp from '../shared/otel-operator-cleanup.md'
 In this tutorial, we would like to introduce [OpenTelemetry Operator][1] which makes
 it very easy to set up Collector and instrument workloads deployed on Kubernetes.
 
-In general, OpenTelemetry deployments and instrumentations is known to be a tedious
-process. An application can be instrumented either automatically or manually.
+In general, OpenTelemetry collector deployments and instrumentations are known to
```
Comment (pranay01) [inline docs/tutorial/instrumentation-using-opentelemetry-operator.md]: I don't think we should not write `OpenTelemetry deployments and instrumentations is known to be a tedious` - we can say something on the lines of "Using Opentelemetry operator will make instrumentation much easier`

File: docs/tutorial/instrumentation-using-opentelemetry-operator.md [hunk 2]
```diff
@@ -185,26 +190,7 @@ spec:
 EOF
 ```
 
----
-To obtain name of the Pet Clinic pod:
```
Comment (pranay01) [inline docs/tutorial/instrumentation-using-opentelemetry-operator.md:189]: any specific reason for removing Pet Clinic example?

Comment (prashant-shahi) [reply docs/tutorial/instrumentation-using-opentelemetry-operator.md:189]: it is redundant part, which was added there by mistake.

File: docs/tutorial/instrumentation-using-opentelemetry-operator.md (hunk 1 above)
Comment (prashant-shahi) [reply docs/tutorial/instrumentation-using-opentelemetry-operator.md]: updated

**Pull request merged**